src/views/annual-report/2021/l10n.json

```diff
@@ -0,0 +1,359 @@
+
+    "annualReport.2021.yearInReviewCard5Date": "May",
+    "annualReport.2021.yearInReviewCard5Title": "Black Lives Matter",
+    "annualReport.2021.yearInReviewCard5Text": "As racial justice protests swept the US, the community came together to support each other and call for change.",
+
+    "annualReport.2021.yearInReviewCard6Date": "June",
+    "annualReport.2021.yearInReviewCard6Title": "Fun At Home! Scratch Design Studio",
+    "annualReport.2021.yearInReviewCard6Text": "Scratchers shared their favorite indoor games and activities to keep themselves engaged while staying home.",
+
+    "annualReport.2021.yearInReviewCard7Date": "June",
+    "annualReport.2021.yearInReviewCard7Title": "Juneteenth Studio",
+    "annualReport.2021.yearInReviewCard7Text": "Scratchers created projects to honor Juneteenth and the continued fight for racial justice.",
+
+    "annualReport.2021.yearInReviewCard8Date": "July",
+    "annualReport.2021.yearInReviewCard8Title": "Scratch Camp",
+    "annualReport.2021.yearInReviewCard8Text": "Scratch the Musical got the whole community acting, singing, and dancing together.",
+
+    "annualReport.2021.yearInReviewCard9Date": "October",
+    "annualReport.2021.yearInReviewCard9Title": "Scratchtober",
+    "annualReport.2021.yearInReviewCard9Text": "Scratchers made hundreds of creative stories, games, and animations based on daily themed prompts.",
+
+    "annualReport.2021.communityQuote2Name": "Anna Lytical, Scratch Alum",
+    "annualReport.2021.communityQuote2Title": "Google Cloud Platform Developer Relations Engineer, and the Coding Drag Queen",
+    "annualReport.2021.communityQuote2Text": "Seeing the power you have when you’re creating something and can represent yourself and your problems and express them or solve them with code is a really magical experience and has real world impact.",
+
+    "annualReport.2021.communitySnapshotTitle": "Improving Our Tools",
+    "annualReport.2021.communitySnapshotText": "Our Community Team uses a wide variety of tools and strategies to encourage good digital citizenship and maintain a positive environment for Scratchers to create in. In 2021, we developed a new, more intuitive interface to help Scratchers flag inappropriate content, and improved the tools used by our community moderation team. As a result, we received higher quality reports from the community, and our community moderators were able to work more quickly and efficiently—keeping the site safer and friendlier for everyone.",
+
+    "annualReport.2021.communitySnapshot2Title": "New Scratch Tutorials on YouTube",
+    "annualReport.2021.communitySnapshot2Text": "The Scratch Team began sharing tutorials on our YouTube channel in March 2021 to help Scratchers gain the skills to create whatever they can imagine. From pixel art to virtual pets, these tutorials are a hit with Scratchers of all ages, gaining 1.3 million views in 2021.",
+
+    "annualReport.2021.tutorial1": "Virtual Town",
```

Comment:
Would it be feasible at this point to remove strings copied from the 2020 Annual Report that aren't in the 2021 Annual Report? Since we are planning to merge more incrementally this year, the translators will see any strings that are in this file (which will allow them to translate before launch), and we don't want them to spend time translating string that won't be in the final report.

Reply:
All of new content should be in and all of the old content should no longer be there! I still need to fill in some of the alt text, but it should be ok to start translating now.

Reply:
wow that's awesome, thank you!